google-login-plugin/testSrc/com/google/gct/login/stats/UsageTrackerManagerTest.java

```diff
@@ -0,0 +1,48 @@
+package com.google.gct.login.stats;
+
+import com.intellij.ide.util.PropertiesComponent;
+import junit.framework.TestCase;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import static org.mockito.Mockito.when;
+
+/**
+ * Tests for {@link UsageTrackerManager}.
+ */
+public class UsageTrackerManagerTest extends TestCase {
+    @Mock
+    private PropertiesComponent mockComponent;
+
+    public void testHasUserRecordedTrackingPreference_noPrefSet() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
Instead of calling this explicitly in each method, I think you can just annotate the entire class as

@RunWith(MockitoJUnitRunner.class)